eng/pipelines/repoman-gha.yml

```diff
@@ -0,0 +1,173 @@
+trigger:
+  paths:
+    include: 
+      - generators/
+      - ext/gha/
+      - eng/pipelines/repoman-gha.yml
+  branches: 
+    include:
+      - main
+
+pr:
+  paths:
+    include:
+      - generators/
+      - ext/gha/
+      - eng/pipelines/repoman-gha.yml
+
+stages:
+  - stage: Validation
+    jobs:
+      - job: Generate_Repos_For_PR
+        condition:  >-              
+          and(
+            succeeded(),
+            or(
+              eq(variables['BuildReasonOverride'], 'PullRequest'),
+              and(
+                eq('', variables['BuildReasonOverride']),
+                eq(variables['Build.Reason'], 'PullRequest')
+              )
+            )
+          )
+        pool:
+          name: azsdk-pool-mms-ubuntu-2004-general
+          vmImage: MMSUbuntu20.04
+
+        variables:
+          RepoManResultsFile: $(Pipeline.Workspace)/repoman.md
+
+        steps:
+          - checkout: self
+
+          - pwsh: |
+              $PRNumber = '$(System.PullRequest.PullRequestNumber)'
+              if ($env:PRNUMBEROVERRIDE) {
+                Write-Host "PR Number override: $($env:PRNUMBEROVERRIDE)"
+                $PRNumber = "$($env:PRNUMBEROVERRIDE)"
+              }
+              Write-Host "##vso[task.setvariable variable=PRNumber]$PRNumber"
+            displayName: Set PRNumber
+
+          - template: /eng/pipelines/templates/steps/repoman-generate.yml
+            parameters:
+              TargetBranchName: pr/$(PRNumber)
+              RemoteName: $(StagingRemoteName)
+              ResultsFileLocation: $(RepoManResultsFile)
+
+          - task: PowerShell@2
+            name: DetermineChanged
```

Comment:
We should only check the gh-action repo here.  We might not need the full script that check multiple repos.